docker/alertmanager.yml

```
@@ -6,3 +6,11 @@ receivers:
 - name: "default"
   webhook_configs:
   - url: http://web:8000/api/v1/alerts
+    # Promgen requires authentication for this webhook endpoint.
+    # First, please access to Promgen's Admin site to create a service account specifically
```

This can be a little bit misleading for users, since they may think there is an option to create service accounts.

```suggestion
    # First, please access to Promgen's Admin site to create a user account specifically
```